lib/osf-components/addon/components/placeholder/template.hbs

```diff
@@ -0,0 +1 @@
+{{yield (hash circle=(component 'placeholder/circle'))}}
```

Comment:
We will almost certainly expand the placeholder types. Can we start with:
```suggestion
{{yield (hash
    circle=(component 'placeholder/circle')
)}}
```
so additional placeholder types are just new lines?